wallet/src/main/java/ch/admin/bag/covidcertificate/wallet/data/WalletSecureStorage.kt

```diff
@@ -39,4 +41,14 @@ class WalletSecureStorage private constructor(context: Context) {
 		KEY_CERTIFICATE_LIGHT_UPDATEBOARDING_COMPLETED, completed
 	).apply()
 
+	fun getTransferCodePublicKey(keyAlias: String) = prefs.getString(KEY_TRANSFER_CODE_PUBLIC_KEY_PREFIX + keyAlias, null)
+
+	fun setTransferCodePublicKey(keyAlias: String, encodedKey: String?) =
+		prefs.edit().putString(KEY_TRANSFER_CODE_PUBLIC_KEY_PREFIX + keyAlias, encodedKey).apply()
+
+	fun getTransferCodePrivateKey(keyAlias: String) = prefs.getString(KEY_TRANSFER_CODE_PRIVATE_KEY_PREFIX + keyAlias, null)
+
+	fun setTransferCodePrivateKey(keyAlias: String, encodedKey: String?) =
+		prefs.edit().putString(KEY_TRANSFER_CODE_PRIVATE_KEY_PREFIX + keyAlias, encodedKey).apply()
+
```

Comment:
Maybe a comment here warning that these functions are only intended for use on Marshmallow?